tests/test_collection.py

```diff
@@ -14,119 +14,131 @@
 from tests.helpers import assert_raises, extract, generate_col_name
 
 
-def test_collection_attributes(db, col, username):
+def test_collection_attributes(client, col, sys_fabric):
```

Comment:
here it should actually be tst_fabric (since to get sys fabric you will need useFabric("_system") ) plus col collection was created in tst_fabric